src/index.html

```diff
@@ -6,9 +6,35 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 class="visually_hidden">Moyo</h1>
+    <header class="header container">
+      <a href="#home">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="header_nav">
+        <ul class="header_nav__list">
+          <li class="header_nav__item" ><a class="header_nav_link is-active" href="#">Apple</a></li>
+          <li class="header_nav__item" ><a class="header_nav_link" href="#">Samsung</a></li>
+          <li class="header_nav__item" ><a class="header_nav_link " href="#">Smartphones</a></li>
+          <li class="header_nav__item" >
+            <a class="header_nav_link " href="#"
+              data-qa="hover" >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header_nav__item" ><a class="header_nav_link " href="#">Gadgets</a></li>
+          <li class="header_nav__item" ><a class="header_nav_link " href="#">Tablets</a></li>
+          <li class="header_nav__item" ><a class="header_nav_link " href="#">Photo</a></li>
+          <li class="header_nav__item" ><a class="header_nav_link " href="#">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
   </body>
```

Comment:
```suggestion
    </header>
  </body>
```
Don't leave empty lines between parent and child elements